ViewModelKit/VMKTableViewRowActionCreator.m

```diff
@@ -21,12 +21,61 @@ - (instancetype)initWithTableView:(UITableView *)tableView rowActionsType:(id<VM
     return self;
 }
 
+#pragma mark - UISwipeActionsConfiguration
+
+- (nullable UISwipeActionsConfiguration *)swipeActionsConfiguration {
+    
+    NSMutableArray<UIContextualAction *> * actions = [NSMutableArray array];
+    
+    for (VMKTableViewRowActionViewModel *rowAction in self.rowActionsType.rowActions) {
+        
+        UIContextualActionStyle style = [self contextualActionStyleFromRowActionViewModel: rowAction];
+        
+        UIContextualAction *tableViewAction = [UIContextualAction contextualActionWithStyle: style
+                                                                                      title: rowAction.title handler:^(UIContextualAction * _Nonnull action, __kindof UIView * _Nonnull sourceView, void (^ _Nonnull completionHandler)(BOOL)) {
+            VMKViewModel *viewModel = [rowAction swipedRowAction];
+            if (viewModel) {
+                // positions were voided
+                VMKTableViewDataSource *tableViewDataSource = self.tableView.dataSource;
+                BOOL shouldOverrideSourceViewForSwipeAction = [rowAction.delegate shouldOverrideSourceViewForSwipeAction];
+                UIView * cellView = sourceView.superview.superview;
```

Comment:
My suggestion here is to iterate up through the view hierarchy for accessing the cell view to avoid the issues in future if there will be any changes in hierarchy. Please refer the example 
`UIView *view = sourceView;
while (view && ![view isKindOfClass:[UITableViewCell class]]) { 
    view = view.superview;
}
UITableView *tableView = view;
UIView *cellView = tableView.contentView;  
`

Reply:
So view hierarchy is following:
```
---- 0 <UISwipeActionPullView: 0x6160001c3880; cellEdge = UIRectEdgeRight, actions = <NSArray: 0x604000137890>>
---- 1 <_UITableViewCellSwipeContainerView: 0x61400007fc40; frame = (0 111; 375 45); clipsToBounds = YES; autoresizesSubviews = NO; layer = <CALayer: 0x6030005f5900>>
---- 2 <UITableView: 0x61f00005e880; frame = (0 0; 375 630);
...
till UIWindow
```
There is neither UITableViewCell nor cell.contentView. It's only **_UITableViewCellSwipeContainerView** which has frame which is close to cell's frame.
So we're not fully protected from the API changes here